### DISCUSSION TITLE
[ 功能建议 ] 能否支持大批量的数据导出功能

### DISCUSSION THREAD

**Discussion by 402309718** [hhyo/Archery#1598] [General]
在提交 issue 前，请查阅以下资源，请先进行搜索来保证没有类似的 issue。
[文档](https://github.com/hhyo/archery/wiki) | [FAQ](https://github.com/hhyo/archery/wiki/FAQ) 

### 重现步骤
1.执行一个结果集较大查询大约4w+数据
2.使用table的导出成excel或者txt，页面hang死，无响应

### 期待结果和实际结果
期待结果：
后台导出

实际结果:
前台导出
### 版本信息
应用版本/分支：

部署方式：Docker、手工部署

**Comment by hhyo** [comment]
大批量的数据导出暂时不考虑支持，查询平台目前的定位是辅助排查问题，当然如果有好的解决方案欢迎提交pr

**Comment by dongqianzheng** [comment]
也不是无响应，就是数据量稍大点就会很慢，等个几分钟还是能导出的。

**Comment by dongqianzheng** [comment]
我们目前是把查询平台当作了唯一的查询入口，开发只能从平台查数据和导出数据。这样就会有一些问题，比如最多只能导出2000行。

**Comment by dba3306** [comment]
是的，我们也遇到类似的问题，大数据量的话是走人工的。

**Comment by hbbdgyb** [comment]
请问如何关闭导出功能？

**Comment by Aaron199** [comment]
> 是的，我们也遇到类似的问题，大数据量的话是走人工的。

我也遇到类似的问题，希望官方能支持，谢谢
